packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -107,46 +115,22 @@ Accessibility recommendations:
 - Any component can be placed in a Table, but tables should only contain static textual and numeric data, rather
   than actionable components like buttons, whenever possible. This is because, if the table is reasonably long—say
   50 rows—with multiple actions per row—say 4 actions—a keyboard-only user would have to press the tab key 200
-  times to get past that table.
+  times to get past that table. If you do need to include interactive elements (such as linking to object details,
+  selecting rows, or deleting rows), use <SiteLink to="/components/data-grid">Data Grid</SiteLink> instead.
 
 <Callout>
-  <CalloutTitle>🦮 Accessibility insight: navigating tables with a screen reader</CalloutTitle>
+  <CalloutTitle>
+    🦮 Accessibility insight: navigating tables with a screen reader
```

Comment:
Tiny note: I've noticed we have been removing emojis from docs pages, should this be removed too?

Reply:
Good call, just removed that!